packages/core/src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ComponentProps } from 'react';
+import { Chip as MaterialChip, makeStyles } from '@material-ui/core';
+
+import { BackstageTheme } from '@backstage/theme';
+
+type Props = ComponentProps<typeof MaterialChip>;
+
+const isSmall = (size: Props['size']) => size === 'small';
+
+const useStyles = makeStyles<BackstageTheme, { size: Props['size'] }>(
+  theme => ({
+    chip: {
```

Comment:
Is it possible to get away with extending https://github.com/spotify/backstage/blob/1812fd05e6cc27cb1c845020b422d6e82b330183/packages/theme/src/baseTheme.ts#L193-L199 instead of wrapping the MUI component?